homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -228,6 +229,9 @@ def __init__(
             model=sensor.model,
         )
         self.index = index
+        self._attr_suggested_display_precision = (
```

Comment:
Why does it ever need to be None. Just always set it at 2.

Reply:
Right. Now that you mentioned that. Just use the entity description to set this field when relevant